src/main/webapp/app/iris/overview/base-chatbot/iris-base-chatbot.component.ts

```diff
@@ -521,22 +521,18 @@ export class IrisBaseChatbotComponent implements OnInit, OnDestroy, AfterViewIni
             rangeStartDate.setHours(0, 0, 0, 0); // Set to the start of the 'daysAgoOlder' day
         }
 
-        return this.chatSessions
-            .filter((session) => {
-                const sessionCreationDate = new Date(session.creationDate);
+        return this.chatSessions.filter((session) => {
+            const sessionCreationDate = new Date(session.creationDate);
 
-                const isAfterOrOnStartDate = ignoreOlderBoundary || (rangeStartDate && sessionCreationDate.getTime() >= rangeStartDate.getTime());
-                const isBeforeOrOnEndDate = sessionCreationDate.getTime() <= rangeEndDate.getTime();
+            const isAfterOrOnStartDate = ignoreOlderBoundary || (rangeStartDate && sessionCreationDate.getTime() >= rangeStartDate.getTime());
+            const isBeforeOrOnEndDate = sessionCreationDate.getTime() <= rangeEndDate.getTime();
 
-                if (ignoreOlderBoundary) {
-                    return isBeforeOrOnEndDate;
-                } else {
-                    return isAfterOrOnStartDate && isBeforeOrOnEndDate;
-                }
-            })
-            .toSorted((a, b) => {
-                return new Date(b.creationDate).getTime() - new Date(a.creationDate).getTime();
-            });
```

Comment:
I only removed the sorting here, we do not want to sort client side as this is expensive!

We are already returning the entries sorted from the server